.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: docker
```

Comment:
Please rename the file to `publish.yml` and also the name:

```yaml
name: publish
```

Reply:
done